docs/references/beta-only-features.md

```diff
@@ -79,8 +79,15 @@ feature.
 
 ## Stabilizing a feature
 
-    TODO: How to move a feature from beta-only to building unconditionally.
+Careful scoping of [defined features](../../common/config/babel/.babelrc.js) will allow you to easily stabilize the feature. When you are ready to add your feature to the stable build:
 
+- One-step stabilization: Include all associated conditionally compiled code in the stable flavored build by moving your feature to `stabilizedFeature` in the [defined features config file]((../../common/config/babel/.babelrc.js)).
+  - Switch to stable flavor and build, test and run samples to try out your stabilized feature.
+  - You will also get the updated API files for the stable build from this step. You can use this for API review.
+- Clean up: Once your feature is shipped in a stable release, remove all references to conditional compilation directive for your feature, and remove your feature from `stabilizedFeature` list.
+  - Note: If some code has other conditional compilation directives along with your (now stable) feature, remove all directives (not only yours), so that the code is included in the stable flavor build unconditionally.
```

Comment:
How would this be solved by `stabilizedFeature` 
Is it intelligent enough to work around this?

For example

// @conditional-remove(bla bla)
function abc() {
  // @conditional-remove(cha cha)
  doSomethingExtra();
}

Reply:
Oh, perhaps I should reword this. I meant same AST node with different features. e.g.:

```ts
export interface StableArgs {

  // @conditional-compile-remove(some-feature-that-needs-options)
  // @conditional-compile-remove(another-feature-that-needs-options)
  options: Options;
}

// @conditional-compile-remove(some-feature-that-needs-options)
// @conditional-compile-remove(another-feature-that-needs-options)
export interface Options {
  // @conditional-compile-remove(some-feature-that-needs-options)
  myFancyOption: boolean;
  // @conditional-compile-remove(another-feature-that-needs-options)
  yourFancyOption: boolean;
}
```

In this example, either feature can be stabilized first. In each case, `Options` will be included in the stable build, but only one of the two fields will be included (depending on which feature is being stabilized).

Perhaps I should add this example to the doc.

Reply:
Added to live example.